docs/organizations/integrations/slack-integration.md

```diff
@@ -0,0 +1,25 @@
+# Organization Slack integration for Security issues
+
+This integration sends notifications to a Slack channel of your choice whenever Codacy detects new critical Security issues in any repository within your organization.
```

Comment:
Using "This integration" kind of forces the reader to look at the title again to confirm the integration. (I did it! 😉)
Maybe we can use a "complete" sentence, as we do on other similar introductions (check [this one](https://tarot-2301-doc-document-slack-integ--docs-codacy.netlify.app/organizations/configuring-default-git-provider-integration-settings/), or [this one](https://tarot-2301-doc-document-slack-integ--docs-codacy.netlify.app/organizations/using-coding-standards/)).